marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -543,6 +596,18 @@ propLastChainPointOnRewindedIndexer = property $ do
     result <- liftIO $ raiseException $ Storable.query Storable.QEverything indexer'' LastSyncPoint
     result === LastSyncPointResult lastestPointPostRollback
 
-getSlot :: C.ChainPoint -> Maybe C.SlotNo
-getSlot C.ChainPointAtGenesis = Nothing
-getSlot (C.ChainPoint s _)    = Just s
+-- | make unique queries
+mkUtxoQueries
```

Comment:
I think this function is not useful outside of propUtxoQueryByAddressAndSlotInterval, right? Should it go inside a `where` clause?